docs/API.md

```diff
@@ -230,11 +230,9 @@ One of the strings (in order from least to most severe):
 
 Basic data type: string
 
-Either:
- * A string of at least 1 and at most 9 digits where the first digit is not a zero, or
- * a string of exactly 16 lower-case hex-digits.
+ * A string of exactly 16 lower-case hex-digits.
```

Comment:
Could you remove the asterisk for consistency with the other datatype specifications>

Reply:
Done.